code/modules/reagents/oldchem/reagents/reagents_med.dm

```diff
@@ -110,11 +110,11 @@
 /datum/reagent/cryoxadone/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
 	if(M.bodytemperature < 265)
-		M.adjustCloneLoss(-4)
```

Comment:
Cryox is the one way of curing clone damage, besides like one chem that requires carp toxin to get and has nasty side effects (rezadone) - perhaps leave that value where it is.